packages/lib-classifier/src/plugins/tasks/TextTask/models/TextAnnotation.js

```diff
@@ -0,0 +1,8 @@
+import { types } from 'mobx-state-tree'
+
+const TextAnnotation = types.model('TextAnnotation', {
+  task: types.identifier,
```

Comment:
There's a general `Annotation` model that sets this and the other task types are composed with general model. I still see value in using composition to make sure we don't repeat ourselves. What do you think of continuing this with the plugin architecture? Since it's just this one part of the model, I don't have a strong opinion to continue this, but just want to get thoughts on it.

Reply:
Composition is definitely the way to go. If the task code moves, I think the shared code has to go with it.

I’ve started adding common views and actions to the base task model in a couple of other PRs.